static/templates/confirm_dialog/confirm_deactivate_user.hbs

```diff
@@ -1,7 +1,29 @@
 <p>
     {{#tr}}
         When you deactivate <z-user></z-user>, they will be immediately logged out.
-        {{#*inline "z-user"}}<strong>{{username}}</strong>{{#if email}} &lt;{{email}}&gt;{{/if}}{{/inline}}
+        {{#*inline "z-user"}}<strong>{{username}}</strong>{{#if email}} &#40;{{email}}&#41;{{/if}}{{/inline}}
```

Comment:
Why these changes?

Reply:
I used round brackets when the user email was still displayed in the text of the checkbox, because i think that those make the next look better and then i adapted those, so they would look the same.
But i changed them back